ux/designs/integration_details/integration_details_page.md

```diff
@@ -0,0 +1,115 @@
+
+1. If there’s no draft for an integration. The draft section would not be shown on this page.
+
+2. Users would be able to “Create Draft” from the current version.
+
+#### Basic layout
+
+![basic layout](img/integration_details_page_rework_11.16_genericlayout.png)
+
+1. Moved status label to the top section right next to Integration Name. Using “check mark” icon to indicate this integration is active.
+
+2. Moving Draft out of the integration history. Users would be able to “Publish” and “Edit” the draft.
+
+3. Added row indicator (green bar) and checkmark icon to indicate active and running version. For active version, users would only be able to perform “Replace Draft” with the active version.
+
+![many steps and connections](img/integration_details_page_rework_11.16_many_stepsconnections.png)
+
+1. Showing an integration that has many connections and steps.
+
+#### Expanding a row
+
+![expandable row and scrolling](img/integration_details_page_rework_11.16_expand_scroll.png)
+
+1. When expanding a row item, show more details about that version.
+
+  - Start Date/Time
+  - Las Run
+  - Last Modified: user name and timestamp
+  - Modification Notes: works like commit message or note to self
+
+2. Users are able to scroll on this page. When scrolling, the whole page scrolls.
+
+
+### Interacting with the page
+
+#### Kebab menu
+
+![kebab menu](img/integration_details_page_rework_11.16_kebab.png)
+
+1. Breadcrumb & toolbar level kebab, users are able to:
+
+  - Stop Integration
+  - Replace Draft
+  - View Logs
+  - Delete Integration (all versions)
+
+  When selecting View Logs, users would be directed to the Logs page.  
+
+2. Row level kebab menu includes:
+
+  - Replace Draft
+  - Publish
+
+
+#### Modal for confirming actions
+
+![replace draft modal](img/integration_details_page_rework_11.16_replaceadraft.png)
+
+Once users click on “Replace Draft”, display a modal to notify users and to confirm the replace draft action.
+
+
+![publish version modal](img/integration_details_page_rework_11.16_publishaversion.png)
+
+If users click on “Publish”, display a modal to notify users what would happen and ask them to confirm their action.
+
+
+#### Status indicators
+
+![spinning up](img/integration_details_page_rework_11.16_spinning_up.png)
+
+1. In between state - showing the current publish version is still active, and a new version is spinning up.
+
+2. The version spinning up would have a spinner indicator.
+
+![being published](img/integration_details_page_rework_11.16_being_published.png)
+
+In this case, the published version is being “decommissioned” and the new version is being published.
+
+1. Show spinner for overall integration status.
+
+2. Show spinners for both the “decommissioning” version and the being published version.
+
+
+#### Notifications (alerts)
+
+![warning message](img/integration_details_page_rework_11.16_warning.png)
+
+1. Display a warning message when a published integration is outdated.
+
```

Comment:
What actually means "outdated" ? As you can change a revision (you can only copy it over as draft), there is nothing like an revision being outdated imo.

Reply:
This design is to satisfy the use case when there are updates available for an integration - 
Examples (from Hiram) - API client connector and technical extensions for example. We should perhaps change the wording of this message. @TovaCohen please advise. I think something like "Updates are available for this integration." might work? And that we should probably avoid using "outdated" as a term.

Reply:
How about: "Updates are available for one or more connections or steps in the active integration. Without these updates, your integration might not behave as expected."
I'm a little concerned about referring to the "published" integration because every integration listed in the history has been published. But there is only one "active" integration.

Reply:
That's great @TovaCohen and thank you for the quick turnaround.
I wonder if it's too long for the space allotted in the UI. @dongniwang could try this text to see what it looks like, but maybe we could use a slightly edited version?

Reply:
Is this short enough:
"An update is available for a connection or step. Republish to ensure correct behavior."
I think that saying that an update is available for the integration is just a little too vague. And I think that "An update" is fine even if there is more than one update, since it doesn't matter how many there are.

Reply:
Also, because of the location of this message, I think it is obvious that it applies to the active integration. So the message does not need to refer to the integration for which updates are available.

Reply:
Thanks @TovaCohen! I'll go with "An update is available for a connection or step. Republish to ensure correct behavior." It's a little vague and generic. That's why I think we need the "more information" link if users want to find out what those updates are.